pages.fa/common/uname.md

```diff
@@ -0,0 +1,20 @@
+# uname
+
+> نمایش اطلاعاتی درباره سخت افزار و سیستم عامل.
+> نکته: برای دستیابی به اطلاعات اضافه در رابطه با سیستم عامل از دستور 'lsb_release' استفاده کنید.
+
+- نمایش اطلاعات مربوط به سخت افزار و پردازنده سیستم:
+
+`uname -mp`
+
+- نمایش اطلاعات مربوط به نرم افزار از جمله: سیستم عامل، شماره انتشار و نسخه:
+
+`uname -srv`
+
+- نمایش نام سیستم:
+
+`uname -n`
+
+- نمایش تمامی اطلاعات سیستم)سخت افزار، نرم افزار، نام سیستم(:
```

Comment:
@opoet777 There's two `(` open parenthesis characters and no `)` closing parenthesis character right here...is that correct in Persian?

Reply:
Yes, I guess this is a problem in github editor for RTL languages.
Actually i open and close parenthesis in right order.

Reply:
So that's right what you did?

Reply:
In github editor is not correct, but i created a specific client for persian language, i think it should be correct in this client.
https://opoet777.github.io/tldr-persian/

Reply:
Wow!  It's really nice!  I'm glad more users can use tldr-pages! :heart:

Since this PR is not merged yet, I tried adding that text with the parenthesis to a page in your client with Chrome's developer tools, and this is what it looks like:

<img width="464" alt="Screen Shot 2021-01-24 at 10 17 51 AM" src="https://user-images.githubusercontent.com/54780737/105634800-dd0ba800-5e2d-11eb-888f-4c13652aa7ee.png">

Does that look right?

Reply:
In RTL languages, do the parenthesis look like `) (` instead of `( )` as they do in LTR languages?

Reply:
No.
This is correct.
![IMG_20210124_190038](https://user-images.githubusercontent.com/56496286/105635102-81b1d800-5e76-11eb-9a8d-0a1d0a1a5352.jpg)

Reply:
Ah okay, that fixed it! Nice job 👍🏻